src/Entity/Article.php

```diff
@@ -1,40 +1,40 @@
 <?php
 
-namespace Afsy\Bundle\CoreBundle\Entity;
+namespace Afsy\Entity;
 
-use Doctrine\ORM\Mapping as ORM;
 use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use DoctrineExtensions\Taggable\Taggable;
 use Gedmo\Mapping\Annotation as Gedmo;
 use Symfony\Component\Validator\Constraints as Assert;
-use DoctrineExtensions\Taggable\Taggable;
 
 /**
  * Article model.
  *
  * @author Xavier LACOT <xlacot@jolicode.com>
- * @ORM\Entity(repositoryClass="Afsy\Bundle\CoreBundle\Entity\ArticleRepository")
+ * @ORM\Entity(repositoryClass="Afsy\Repository\ArticleRepository")
  */
 class Article implements Taggable
 {
     /**
      * @ORM\Id
      * @ORM\Column(type="integer")
-     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\GeneratedValue
      */
     protected $id;
 
     /**
      * @var string
-     * @ORM\Column(type="string", length=255)
```

Comment:
Pourquoi avoir enlever le type ?

Reply:
C’est le type par défaut ainsi que la longueur par défaut. C’est du bruit !

Reply:
Dans ce cas la autant virer les parenthèses inutiles et les accolades sur les if for while etc :D

Reply:
Explicit is better than implicit 
https://www.python.org/dev/peps/pep-0020/